website/content/docs/agent-and-proxy/agent/caching/index.mdx

```diff
@@ -7,18 +7,23 @@ description: |-
   newly created tokens.
 ---
 
-# Vault agent caching
+# Vault Agent caching
+
+<Note title="Use Vault Proxy for static secret caching">
+
+  [Static secret caching](/vault/docs/agent-and-proxy/proxy/caching/static-secret-caching)
+  (KVv1 and KVv2) with API proxy minimizes the number of requests forwarded to
+  Vault. Vault Agent does not support static secret caching with API proxy. We
+  recommend using [Vault Proxy](/vault/docs/agent-and-proxy/proxy) for API Proxy
+  related workflows.
+
+</Note>
 
 Vault Agent Caching allows client-side caching of responses containing newly
```

Comment:
Are we calling this "caching" or "persistence?" (this is very pedantic, but I just want to be clear).

Reply:
Vault Agent does have caching for dynamic secrets today. While we don't encourage its use, it does still support the API proxy!

Reply:
Ah this is a bit confusing. I'm fine with this at this time, but let's look at the docs holistically in our next 1-1 and see if we can come up with some ideas